master-firmware/src/robot_helpers/trajectory_helpers.c

```diff
@@ -63,7 +63,7 @@ int trajectory_has_ended(int watched_end_reasons)
         if (messagebus_topic_read(proximity_beacon_topic, &beacon_signal, sizeof(beacon_signal)) && timestamp_duration_s(beacon_signal.timestamp.us, timestamp_get()) < TRAJ_MAX_TIME_DELAY_OPPONENT_DETECTION && beacon_signal.range.range.distance < TRAJ_MIN_DISTANCE_TO_OPPONENT) {
             float x_opp, y_opp;
             beacon_cartesian_convert(&robot.pos,
-                                     1000 * beacon_signal.range.range.distance,
+                                     1.25 * 1000 * beacon_signal.range.range.distance,
```

Comment:
This correction should be done somewhere else.
Also, the error is an offset not a factor.

Reply:
euh, this is not what I wanted to do :man_facepalming: 
I need to scale the opponent obstacle size here